Gems/Terrain/Assets/Shaders/Terrain/TerrainSrg.azsli

```diff
@@ -149,6 +157,25 @@ ShaderResourceGroup TerrainSrg : SRG_Terrain
     {
         return uint2(m_macroMaterialGrid.m_resolution >> 16, m_macroMaterialGrid.m_resolution & 0xFFFF);
     }
+    
+    float2 CalculateCullDistance(uint lodLevel, float3 worldPosition)
+    {
+        float minRange = 0;
+        float maxRange = TerrainSrg::m_meshData.m_firstLodDistance;
```

Comment:
You don't need the TerrainSrg:: here
(and you don't use it three lines down, so the inconsistency was slightly confusing, I thought there were two different m_meshData for a moment)

Reply:
Ah good catch - I moved this code from outside the SRG to inside it so I could unify it across forward / shadow / depth. Missed removing that TerrainSrg reference.